test/helpers/agent.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+// eslint-disable-next-line node/no-extraneous-require
+const _ = require('lodash');
+// eslint-disable-next-line node/no-extraneous-require
+const qs = require('qs');
+const request = require('supertest');
+const { createUtils } = require('./utils');
+
+const createAgent = (strapi, initialState = {}) => {
+  const _state = initialState;
```

Comment:
the underscore are unnecessary. Using them is necessary if they are exposed publicly to signify they shouldn't be used :) here the _state is in the closure and not exposed. same for _utils & _agent. Moreover agent is returned